kratos/python_scripts/print_info_in_file_process.py

```diff
@@ -0,0 +1,131 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return PrintInfoInFileProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class PrintInfoInFileProcess(KratosMultiphysics.Process):
```

Comment:
This should derive from `OutputProcess`.

Reply:
done